xAPI.md

```diff
@@ -4539,6 +4540,13 @@ property MUST occur only once.
 * An LRS MUST NOT reject requests based on their version header as would otherwise be 
 required by <a href="#versioning"/>Versioning</a>.
 
+<a name="encoding"/> 
+
+### 2.3 Encoding
+
+###### Requirement
+* All strings MUST be encoded and interpreted as UTF-8. 
```

Comment:
I'm not going to do this one as part of this PR, but if you want to do it as a separate PR (perhaps branched off garemoko:issue-798-1-4 or just done from scratch) I would +1 it.